extension/storage/dbstorage/client.go

```diff
@@ -17,10 +17,11 @@ import (
 )
 
 const (
-	createTable     = "create table if not exists %s (key text primary key, value blob)"
-	getQueryText    = "select value from %s where key=?"
-	setQueryText    = "insert into %s(key, value) values(?,?) on conflict(key) do update set value=?"
-	deleteQueryText = "delete from %s where key=?"
+	createTableSqlite = "create table if not exists %s (key text primary key, value blob)"
+	createTable       = "create table if not exists %s (key text primary key, value text)"
```

Comment:
Why text? The getters/setters on the storage extension interface work on `[]byte` type... Doesn't Postgres have a blob type? Maybe it's called differently?

Reply:
postgres has no blob type, a bytea type that is not available by default, and text which works well across dbs. I picked text as it is closest.